Mail/SecurityOtpMail.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace WebReinvent\VaahCms\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class SecurityOtpMail extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    public $content;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct($content)
+    {
+        $this->content = $content;
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        return $this->subject(config('app.name').': Two Step Authentication')
```

Comment:
@we-vikram-wri231 did you check other otp email for `subject` reference?

Reply:
@themodernpk sir, yes I have taken reference from DocuSign and Stripe two factor mail

Reply:
@we-vikram-wri231 subject should match with the message we're showing where they need to enter the OTP